frontend/src/pages/modelRegistry/screens/EmptyRegisteredModels.tsx

```diff
@@ -22,6 +24,18 @@ const EmptyRegisteredModels: React.FC<EmptyRegisteredModelsType> = ({ preferredM
       <br />
       registry or select a different one.
     </EmptyStateBody>
+    <EmptyStateFooter>
+      <Button
+        id="register-model-empty-button"
+        key="register-model-empty-button"
+        data-testid="register-model-empty-button"
+        aria-label="Register model"
+        onClick={() => undefined}
+      >
+        Register model
+      </Button>
+      ,
```

Comment:
need to remove this

Reply:
This was on hold due to das' changes, lready changed